.scripts/restore.cmd

```diff
@@ -34,6 +34,9 @@ ECHO Restoring SDK-Style projects in "%rlRoot%\rl.sln"
 %dotnetPath% restore "%rlRoot%\rl.sln"
 ECHO.
 
+ECHO Restoring DotNet Tools (made simpler in .NET Core 3.0)
```

Comment:
I am curious when does this file run and what does the following statement do?

Reply:
It restores the dotnet-t4 tool, which is a dotnet core (and thus, cross platform) version of the old TextTemplating tool. What this does it enable .NET Core SDK < 3.0 to restore the tool (it has to install it globally, the .json file trigger restoration for local).

This is then used in the new TextTemplating targets file to actually drive the template generation.